### ISSUE TITLE
[Bug Report] Accent of uppercase letters are clipping on v-text-field hint

### ISSUE THREAD

**Issue by peluprvi** [vuetifyjs/vuetify#4655]
### Versions and Environment
**Vuetify:** 1.1.6
**Vue:** 2.5.16
**Browsers:** Chrome 67.0.3396.99
**OS:** Windows 7

### Steps to reproduce
- Open the codepen
- Compare the hint text in javascript with the displayed on the v-text-field

### Expected Behavior
Accents of uppercase letters should shown.

### Actual Behavior
`.v-text-field__details` has `overflow: hidden;` and its height doesn't hold the uppercase letters with accents.

### Reproduction Link
[https://codepen.io/anon/pen/pZEyXd](https://codepen.io/anon/pen/pZEyXd)

**Comment by riggerthegeek** [comment]
I've "fixed" this by adding this to my view. Obviously it's not a proper fix, but it seems like that a PR with this being added would solve it

```
<style lang="scss">
  .v-messages__message {
    padding: {
      bottom: 5px;
    }
  }
</style>
```

**Comment by AlexNigl** [comment]
line-height: 1; is the culprit.
The reason is long and complicated and basically boils down to:
[The font-size probably is not calculated the way you think it is calculated.](https://iamvdo.me/en/blog/css-font-metrics-line-height-and-vertical-align) 

My solution for this particular problem is to set the line-height to 1.33333. The reason for this number is twofold:
1. It doesn't cut any letters in my testcase: "ÄÖÜygq" (Tested in Chrome70 and FF63)
2. With a font-size of 12px (standard) this adds exactly 4px (font-size * line-height) to the box which goes perfect with the [4dp grid of the material design reference](https://material.io/design/layout/spacing-methods.html#baseline)

tldr:
I use:
.v-messages__message {
line-height: 1.33333; 
}

**Comment by vuoriliikaluoma** [comment]
> My solution for this particular problem is to set the line-height to 1.33333. The reason for this number is twofold:
> 
> 1. It doesn't cut any letters in my testcase: "ÄÖÜygq" (Tested in Chrome70 and FF63)
> 2. With a font-size of 12px (standard) this adds exactly 4px (font-size * line-height) to the box which goes perfect with the [4dp grid of the material design reference](https://material.io/design/layout/spacing-methods.html#baseline)

To fit letters like "ÅŮ" That would need to be more like 1.5.
_The Github font isn't rendering that correctly for me, the rings should be at the same height._

This seems to be a pretty complete list of all "above" diacritics: https://www.compart.com/en/unicode/combining/230

I think the most common "above" diacritics are these:
* À - Grave
* Á - Acute
* Â - Circumflex
* Ä - Diaeresis (Umlaut)
* Å - Ring
* Ã - Tilde
* Ǎ - Caron

And 1.5 works with all of them.

**Comment by nekosaur** [comment]
Not reproducible on latest stable (1.5.14) or upcoming 2.0
